### DISCUSSION TITLE
Configure VM guest?

### DISCUSSION THREAD

**Discussion by MScottBlake** [utmapp/UTM#4272] [UTM Help (macOS)]
I am wondering if there is a way to configure a VM once it is created. For instance, if you feed an IPSW file to create a VM, it completes and boots it to the Setup Assistant. Is there a way to automate the setup and maybe create users/install a bootstrap application or something along those lines?

**Comment by MScottBlake** [comment]
I guess I should assume the silence means no?

**Reply by ktprograms** [reply]
I don't think there's any way other than the stuff built in to the OS (enterprise management for macOS).

You could just setup one VM then clone it multiple times.
